README.md

```diff
@@ -1,13 +1,42 @@
-# Builder Image Azure Terraform
+## What is the image for?
+The intended purpose of this image is for it to be used as a Jenkins agent. By using the installed features the user is able to create Jenkins pipelines that can trigger Terraform scripts to deploy to an Azure environment. Additionally, Terragrunt gives the ability to use extra tools for Terraform while TFLint gives us a Terraform linter. An example of using this image as a Jenkins agent via [Kubernetes](https://plugins.jenkins.io/kubernetes/) can be seen below. 
 
-Image for applying a Terraform plan to an Azure environment.
+First, an example of configuring the pod template in yaml to create the agent.
 
-## Tools
+```yaml
+jenkins:
+  clouds:
+    - kubernetes:
+        name: "kubernetes"
+        templates:
+          - name: "image-builder-azure-terraform"
+            label: "image-builder-azure-terraform"
+            nodeUsageMode: NORMAL
+            containers:
+              - name: "image-azure-terraform"
+                image: "ghcr.io/liatrio/image-builder-azure-terraform:${builder_images_version}"
+```
+And then specifying the agent in the Jenkinsfile for an example step.
 
-- Azure CLI
-- GO: 1.18.2
-- YQ: 4.5.1
-- TFLINT: v0.36.2
-- TERRAFORM: 1.2.1
-- TERRAGRUNT: v0.37.1
+```jenkins
+stage('Build') {
+  agent {
+    label "image-builder-azure-terraform"
+  }
+  steps {
+    container('image-azure-terraform') {
+      sh "terragrunt plan"
+      sh "terragrunt apply"
+      sh  'az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET -t $AZURE_TENANT_ID'
```

Comment:
I think for the sake of consistency we should use single quotes on lines 28 and 29. Also there's a double space on line 30.